src/seedu/addressbook/commands/AddAndList.java

```diff
@@ -0,0 +1,75 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
```

Comment:
Imported classes should always be listed explicitly. E.g. `java.util.List` and not `java.util.*`, which does not follow NUS Java coding standards.